extensions/last-commit-info/src/test/resources/jar-last-build-date-info.txt

```diff
@@ -0,0 +1 @@
+2023-05-08T15:30:00Z
```

Comment:
this date is the same one as the other one, you should use two distinct dates to make the test actually test that you put the right thing in the right field.